src/algorithms/tracking/OrthogonalTrackSeedingConfig.h

```diff
@@ -1,102 +1,168 @@
-	maxScatteringAngle * maxScatteringAngle;
-
-      // Helix radius in homogeneous magnetic field
-      // in ACTS Units are GeV, mm, and GeV/(e*mm)
-      m_seedFinderConfig.pTPerHelixRadius = m_seedFinderConfig.bFieldInZ;
-      m_seedFinderConfig.minHelixDiameter2 =
-	std::pow(m_seedFinderConfig.minPt * 2 /
-		 m_seedFinderConfig.pTPerHelixRadius,
-		 2);
-
-      m_seedFinderConfig.pT2perRadius =
-	std::pow(
-          m_seedFinderConfig.highland / m_seedFinderConfig.pTPerHelixRadius,
-	  2);
-
-    }
-
-  };
-
+// Created by Dmitry Romanov
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#pragma once
+
+#include <vector>
+
+#include "Acts/Seeding/SeedFilterConfig.hpp"
+#include "Acts/Seeding/SeedFinderOrthogonalConfig.hpp"
+#include "Acts/Seeding/SeedFilter.hpp"
+#include "Acts/Seeding/SeedConfirmationRangeConfig.hpp"
+
+#include "SpacePoint.h"
+
+namespace eicrecon {
+
+  struct OrthogonalTrackSeedingConfig {
+    //////////////////////////////////////////////////////////////////////////
+    /// SEED FINDER GENERAL PARAMETERS
+    float m_rMax = 440. * Acts::UnitConstants::mm; // max r to look for hits to compose seeds
+    float m_rMin = 33. * Acts::UnitConstants::mm; // min r to look for hits to compose seeds
+    float m_zMax = 1700. * Acts::UnitConstants::mm; // max z to look for hits to compose seeds
+    float m_zMin = -1500. * Acts::UnitConstants::mm; // min z to look for hits to compose seeds
+    float m_deltaRMinTopSP = 10. * Acts::UnitConstants::mm; // Min distance in r between middle and top SP in one seed
+    float m_deltaRMaxTopSP = 200. * Acts::UnitConstants::mm; // Max distance in r between middle and top SP in one seed
+    float m_deltaRMinBottomSP = 10. * Acts::UnitConstants::mm; // Min distance in r between middle and bottom SP in one seed
+    float m_deltaRMaxBottomSP = 200. * Acts::UnitConstants::mm; // Max distance in r between middle and bottom SP in one seed
+    float m_collisionRegionMin = -250 * Acts::UnitConstants::mm; // Min z for primary vertex
+    float m_collisionRegionMax = 250 * Acts::UnitConstants::mm; // Max z for primary vertex
+
+    float m_maxSeedsPerSpM = 0; // max number of seeds a single middle sp can belong to - 1
+    float m_cotThetaMax = 27.29; // Cotangent of max theta angle (27.29 corresponds to eta = 4)
+    float m_sigmaScattering = 5; // How many standard devs of scattering angles to consider
+    float m_radLengthPerSeed = 0.1; // Average radiation lengths of material on the length of a seed
+    float m_minPt = (100. * Acts::UnitConstants::MeV) / m_cotThetaMax; // MeV (in Acts units of GeV) - minimum transverse momentum
+    float m_bFieldInZ = 1.7 * Acts::UnitConstants::T; // T (in Acts units of GeV/[e*mm]) - Magnetic field strength
+    float m_beamPosX = 0; // x offset for beam position
+    float m_beamPosY = 0; // y offset for beam position
+    float m_impactMax = 3. * Acts::UnitConstants::mm; // Maximum transverse PCA allowed

```

Comment:
One thing to consider for the future (definitely not for this PR, as this is already a needed update) - this will (obviously) limit the seeds that are found to be located within 3 mm of the nominal x-y origin. For the vast majority of tracks that is fine, but long lived decays and/or secondaries will never be found. There are two options for future development:
1. deploy iterative tracking which runs an additional seeding pass, either with a re-tuned version of this seeder or some other seeder, to find the missing seeds
2. Continue tuning this set of parameters to allow for larger `m_impactMax`.

I would suggest option (1), but again this is definitely not for this PR and requires additional future development. I just thought I'd mention that the Acts seeder takes this value very seriously

Reply:
Deferred to #648